src/actions/expressions.js

```diff
@@ -22,21 +22,22 @@ function expressionExists(expressions, input) {
  * @memberof actions/pause
  * @static
  */
-function addExpression(input: string) {
+function addExpression(input: string, visible: boolean = true) {
```

Comment:
I've seen secondary options in a trailing options param in some other actions like `addBreakpoint` which might be nice here.

```js
addExpression(input: string, { visible: boolean = true }) {
  // stuff
}
```